packages/components/src/components/accordion/accordion.ts

```diff
@@ -165,6 +164,12 @@ export default class SdAccordion extends SolidElement {
           @click=${this.handleSummaryClick}
           @keydown=${this.handleSummaryKeyDown}
         >
+          <div
+            class=${cx(
+              !this.open && 'opacity-0',
+              'w-1 bg-accent absolute left-0 transition-all h-[75%] group-hover:h-full'
```

Comment:
I don't think that 75% is correct. I believe it should be more an absolute amount of pixels which is removed from the top and the bottom.

E. g.  if the full height is 100px → 75px, if the tull height is 200px → 175px. Otherwise it will stretch strangely in multiline buttons.

Reply:
If this was implemented the same way in `sd-navigation-item` we should adapt it there, too.

Reply:
But please clarify with design to make that sure.